toxicity/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tensorflow-models/toxicity",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
bump to 1.0.0. Can you also make sure the demo uses the latest. Also run yarn to update the yarn.lock file.